src/engine/strat_engine/cmd.rs

```diff
@@ -49,9 +49,46 @@ const UDEVADM: &str = "udevadm";
 const XFS_DB: &str = "xfs_db";
 const XFS_GROWFS: &str = "xfs_growfs";
 const CLEVIS: &str = "clevis";
+const CLEVIS_LIB: &str = "clevis-luks-common-functions";
 const CLEVIS_BIND: &str = "clevis-luks-bind";
 const CLEVIS_UNBIND: &str = "clevis-luks-unbind";
 const CLEVIS_UNLOCK: &str = "clevis-luks-unlock";
+const CLEVIS_DECRYPT: &str = "clevis-decrypt";
+const CLEVIS_DECRYPT_TANG: &str = "clevis-decrypt-tang";
+const CLEVIS_DECRYPT_TPM2: &str = "clevis-decrypt-tpm2";
+const CLEVIS_ENCRYPT: &str = "clevis-encrypt";
+const CLEVIS_ENCRYPT_TANG: &str = "clevis-encrypt-tang";
+const CLEVIS_ENCRYPT_TPM2: &str = "clevis-encrypt-tpm2";
+const JOSE: &str = "jose";
+const JQ: &str = "jq";
+const CRYPTSETUP: &str = "cryptsetup";
+const CURL: &str = "curl";
+const TPM2_CREATEPRIMARY: &str = "tpm2_createprimary";
+const TPM2_UNSEAL: &str = "tpm2_unseal";
+const TPM2_LOAD: &str = "tpm2_load";
+const MKTEMP: &str = "mktemp";
+
+const CLEVIS_EXECS: &[&str] = &[
```

Comment:
Could you put a comment here, indicating how you came up with this list?

Reply:
The way that I came up with this was from the dracut modules that Clevis provides and my development of the dracut modules for booting from Stratis filesystems as the root filesystem. Do you want me to say something to that effect? For example, `mktemp` was not included in the distributed Clevis dracut modules, but that was required for our altered, full install of Clevis whereas it does not appear to be required for the Clevis dracut module architecture because they only add a subset of Clevis functionality.

Reply:
Yes, please do. Even if it needs to be rather long comment, it will be good to have that record right in the source.